plugins/riemann-controller/riemann_controller/resources/manager.config

```diff
@@ -77,7 +77,9 @@
 
 (defn- amqp-connection-conf
   []
-  {:host (or (System/getenv "RABBITMQ_HOST") "localhost")})
+  {:host (or (System/getenv "RABBITMQ_HOST") "localhost")
+   :username (or (System/getenv "RABBITMQ_USER") "testuser")
```

Comment:
Am I missing something? I think all these defaults should be set by the user + centralized. We can't reliably set them across different repos/plugins and expect them to stay synced.